src/__tests__/index.test.ts

```diff
@@ -447,11 +452,9 @@ function expectBlockQueries(versionTime: string) {
 }
 
 async function createCaip10Link(ethAuthProv: EthereumAuthProvider) {
-  const proof = await ethAuthProv.createLink(caipLinkControllerDid);
-  const doc = await ceramic.createDocument('caip10-link', {
-    metadata: { family: 'caip10-link', controllers: [proof.account] }
-  });
-  await doc.change({ content: proof });
+  const accountId = await ethAuthProv.accountId()
+  const link = await Caip10Link.fromAccount((global as any).ceramic, accountId, {})
```

Comment:
```suggestion
  const link = await Caip10Link.fromAccount((global as any).ceramic, accountId, {syncTimeoutSeconds: 0})
```

Reply:
Does this change speed up the tests at all?

Reply:
Yeah, I think this would speed it up. Should be completely fine to do this in the test file.